### DISCUSSION TITLE
Broken right now?

### DISCUSSION THREAD

**Discussion by masonbesmer** [community/community#19003] [Copilot Conversations]
Is Copilot broken for anyone else? My codespace has come to a screeching halt without it. A friend of mine is also experiencing issues.

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by masonbesmer** [comment]
[INFO] [auth] [2022-06-21T20:18:47.192Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-21T20:18:47.194Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

I was invited several months ago and am also a student. Not sure why it says I'm not authorized.

**Comment by BirdMachine** [comment]
Having the same issue today. I've re-authorized and allowed the "Suggestions matching public code" feature, and I get to the success page but when I return to my Billing/Payment page it still asks if I want to join the free trial. Same error message (Guessing also vscode)

**Reply by Sawamuraandmatty** [reply]
Having the same issue.   I recently changed my GitHub password.

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by davecheney** [reply]
look in the bottom left of the vscode window

**Reply by cyaconi** [reply]
Did that but still getting this error:
<img width="441" alt="image" src="https://user-images.githubusercontent.com/91154/174900602-2c0d364c-856c-442e-8f85-82aed1e9ae18.png">
It seems it's something broken right now.

**Reply by MBARIMike** [reply]
I had to restart VS Code after logging out of GitHub before I saw the login prompt from copilot.

**Reply by cyaconi** [reply]
I was able to sign in again. However, besides I have already signed up for the copilot trial (many times, btw), it's showing:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/91154/174901394-df79f824-308e-457e-85b8-713fae68de12.png">

**Reply by henrycunh** [reply]
same here, as with many friends.

**Comment by bronifty** [comment]
You are using co-pilot with a different account. VS Code insists that you use your main account for co-pilot. Try that one.

**Reply by masonbesmer** [reply]
Nope... I am programming in a codespace.
